README.md

```diff
@@ -10,127 +10,99 @@
 
-When using the template you confirm that you have read the [license](LICENSE.md) and comprehend that I can take down
-your repository if you do not meet these requirements.
-
-## How to download it
-
-This repository is now a template, on the top left you can simply click on "**Use this template**" to create a GitHub
-repository based on this template.
-
-Alternatively you can do the following:
-
-- Clone/Download the repository
-  - To clone it and get the updates you can definitely use the command
-    `git clone`
-- Create a discord bot [here](https://discord.com/developers/applications)
-- Get your bot token
+- A basic knowledge of Python.
+- A Discord bot application. You can create one [here](https://discord.com/developers/applications).
 - Invite your bot on servers using the following invite:
   https://discord.com/oauth2/authorize?&client_id=YOUR_APPLICATION_ID_HERE&scope=bot+applications.commands&permissions=PERMISSIONS (
   Replace `YOUR_APPLICATION_ID_HERE` with the application ID and replace `PERMISSIONS` with the required permissions
   your bot needs that it can be get at the bottom of a this
   page https://discord.com/developers/applications/YOUR_APPLICATION_ID_HERE/bot)
 
-## How to set up
+## Setup
 
-To set up the bot it was made as simple as possible.
+1. **Configure `config.json`**:
+   - Set your bot's prefix and invite link in the `config.json` file.
+   - Here is an explanation of what everything is:
+    
+      
+      | Variable                  | What it is                                     |
+      | ------------------------- | ---------------------------------------------- |
+      | YOUR_BOT_PREFIX_HERE      | The prefix you want to use for normal commands |
+      | YOUR_BOT_INVITE_LINK_HERE | The link to invite the bot                     |
+      
 
-### `config.json` file
+2. **Set the Bot Token**:
+   - Create an environment variable named `TOKEN` or edit the `.env` file with your bot token.
+   - To set up the token you will have to either make use of the [`.env.example`](.env.example) file, either copy or rename it to `.env` and replace `YOUR_BOT_TOKEN_HERE` with your bot's token.
 
-There is [`config.json`](config.json) file where you can put the
-needed things to edit.
+3. **Install Requirements**:
+   - Run `python -m pip install -r requirements.txt` to install the required packages.
 
-Here is an explanation of what everything is:
+4. **Start the Bot**:
+   - Run `python bot.py` to launch your bot.
 
-| Variable                  | What it is                                     |
-| ------------------------- | ---------------------------------------------- |
-| YOUR_BOT_PREFIX_HERE      | The prefix you want to use for normal commands |
-| YOUR_BOT_INVITE_LINK_HERE | The link to invite the bot                     |
+## Support
 
-### `.env` file
+If you need help or have questions, feel free to join our [Discord server](https://discord.com/invite/mTBrXyWxAF) for assistance.
```

Comment:
People can also post [issues](https://github.com/kkrypt0nn/Python-Discord-Bot-Template/issues) as long as it's related to the non modified template.